module3/src/models/user.ts

```diff
@@ -0,0 +1,30 @@
+import { DataTypes, Sequelize } from 'sequelize';
+import { v4 as uuid } from 'uuid';
+
+const user = {
+    id: {
+        type: DataTypes.UUIDV4,
+        primaryKey: true,
+        defaultValue: uuid()
+    },
+    login: DataTypes.STRING,
+    password: DataTypes.STRING,
+    age: DataTypes.INTEGER,
+    isdeleted: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: false
+    }
+};
+
+export default (sequelize: Sequelize) => sequelize.define('user', user, {
+    createdAt: false,
+    updatedAt: false
+});
+
+export interface User {
+    id: string,
+    login?: string,
```

Comment:
Can you create user without login or password?

Reply:
Yes, you are right, these fields are required. Interface fixed.